config/database.yml

```diff
@@ -75,23 +9,48 @@ test:
 #
 # On Heroku and other platform providers, you may have a full connection URL
 # available as an environment variable. For example:
-#
 #   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
 #
 # You can use this database configuration with:
-#
 #   production:
 #     url: <%= ENV['DATABASE_URL'] %>
 #
-production:
+#
+# Some less common options:
+# -------------------------
+# Schema search path. The server defaults to $user,public
+#   schema_search_path: myapp,sharedapp,public
+# Minimum log levels, in increasing order (defaults to warning):
+# debug5, debug4, debug3, debug2, debug1, log, notice, warning, error, fatal, and panic
+#   min_messages: notice
+
+default: &default
+  adapter:  postgresql
+  encoding: unicode
+  username: <%= ENV.fetch('POSTGRES_USER',     nil) %>
+  password: <%= ENV.fetch('POSTGRES_PASSWORD', nil) %>
+  database: <%= ENV.fetch('POSTGRES_DB',       nil) %>
+  # https://til.hashrocket.com/posts/6918d4e62b-use-postgresql-socket-in-databaseyml
+  host:     <%= ENV.fetch('POSTGRES_HOST',     '/var/run/postgres') %>
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  pool:     <%= ENV.fetch("RAILS_MAX_THREADS", 5) %>
+
+development:
   <<: *default
-  database: mutualaid_production
-  username: mutualaid
-  password: <%= ENV['POSTGRES_PASSWORD'] %>
+  database: <%= ENV.fetch('POSTGRES_DB', 'mutualaid_development') %>
 
 docker-development:
   <<: *default
-  database: <%= ENV['POSTGRES_DB'] %>
-  username: <%= ENV['POSTGRES_USER'] %>
-  password: <%= ENV['POSTGRES_PASSWORD'] %>
-  host: <%= ENV['POSTGRES_HOST'] %>
+  database: <%= ENV.fetch('POSTGRES_DB', 'mutualaid_development') %>
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: <%= ENV.fetch('POSTGRES_DB', 'mutualaid_test') %>
+
+production:
+  <<: *default
+  database: <%= ENV.fetch('POSTGRES_DB', 'mutualaid_production') %>
+  username: <%= ENV.fetch('POSTGRES_USER', 'mutualaid') %>
```

Comment:
why does this have to be mutualaid in prod, and why wouldn't we add this to default?